packages/grafana-flamegraph/src/FlameGraph/FlameGraphTooltip.tsx

```diff
@@ -141,14 +141,16 @@ export const getDiffTooltipData = (
   const displayValueRight = getValueWithUnit(data, data.valueDisplayProcessor(item.valueRight!));
 
   const shortValFormat = getValueFormat('short');
+  
+  const formatted = shortValFormat(diff);
 
   return [
     {
       rowId: '1',
       label: '% of total',
       baseline: percentageLeft + '%',
       comparison: percentageRight + '%',
-      diff: shortValFormat(diff).text + '%',
+      diff: formatted.text + '%',
```

Comment:
The suffix is a separate property so we still need to add it (though we need to handle if it's not provided):

```suggestion
      diff: `${formatted.text}${formatted.suffix || ''}%`,
```